ezcontent_publish.permissions.yml

```
@@ -0,0 +1,3 @@
+ezcontent_publish view user profile:
```

Can we rename this permission
```suggestion
ezcontent publish view user profile:
```